core/drivers/clk/clk_dt.c

```diff
@@ -13,18 +13,6 @@
 #include <libfdt.h>
 #include <stddef.h>
 
-struct clk *clk_dt_get_by_name(const void *fdt, int nodeoffset,
```

Comment:
> `Move clk_dt_get_by_name() right below clk_dt_get_by_idx().`

Sure, but why? ;-)

Reply:
well... true it looks a bit useless.
The change makes th function easier to read as it calls `clk_dt_get_by_idx()` that is rigth below.

I can discard this change.